src/assets/index.ts

```diff
@@ -0,0 +1,18 @@
+export { default as DocIllustration } from './illustration_doc';
```

Comment:
are these illustrations necessary? I think these are all from the MUI template, and If we don't use them I would not include them in our project.

Reply:
it was required, I would remove it later if will not use it.